node-db/lesson1.md

```diff
@@ -1,32 +1,673 @@
+
+# What is JSON?
+
+JSON is a type of file for structuring data in a readable way. It is also a really popular format for sending data across the web.
+JSON is a string representation of a Javascript object. JSON objects convert really easily to Javascript objects, and vice versa, with `JSON.parse()` and `JSON.stringify()`.
+
+```javascript
+{
+  "firstName": "John",
+  "lastName": "Smith",
+  "isAlive": true,
+  "age": 25,
+  "address": {
+    "streetAddress": "21 2nd Street",
+    "city": "New York",
+    "state": "NY",
+    "postalCode": "10021-3100"
+  },
+  "phoneNumbers": [
+    {
+      "type": "home",
+      "number": "212 555-1234"
+    },
+    {
+      "type": "office",
+      "number": "646 555-4567"
+    },
+    {
+      "type": "mobile",
+      "number": "123 456-7890"
+    }
+  ],
+  "children": [],
+  "spouse": null
+}
+```
+
+# Step 2 - Installing Express
+
+Before we write any code, you'll need to install the Express library. We're going to use the **[Node Package Manager (npm)](https://www.npmjs.com/)** to download it using the **`npm install`** command.
+
+NPM is the place to go to download other Node code written by other people.  There are thousands of open-source, 3rd-party Node modules (also known as "packages") by other people that you can download and use in your own projects. 
+
+As we install Express, we'll need to update the `package.json` to add Express as a dependency. We do this so that other people working on the project will know to install Express before running any of the code. This can be done by adding **`--save`** to the end of your command.
+
+Run the following command in your terminal:
+
+`$ npm install express --save`
+
+Express should now be installed. Check your `package.json` file to make sure it has been added as a dependency. It will look like this:
+
+![package.json screenshot](https://cloud.githubusercontent.com/assets/10683087/16382664/be35f0b4-3c79-11e6-82b6-ae9e4a037c3f.png)
+
+# Step 3 - Building the server
+
+The first thing we need to do is build our server. You will always need to build a server when writing back-end code. A server can be built in pure Node.js, but Express gives us simpler syntax to work with.
+
+
+## 1. Create a `server.js` file
+
+Let's build our server! Before we do anything, let's create a new file called `server.js`. This is where all our server code is going to live.
+
+## 2. `require` the `express` library
+
+We already installed Express in Step 2, but we need to make sure it is included in this file specifically so we can make use of its methods. In Node.js, when you want to access the functionality of a library or module in another file, you `require` it.
+
+To import Express, write the following inside `server.js`:
+
+```js
+const express = require('express');
+```
+
+> We will try and get used to ES6 syntax - so use `const` and `let` instead of `var`.
```

Comment:
I would leave it for a following change. The intention of this PR was just to get to the london branch, what the guys in Scotland have been using in the scotland branch